src/operations/executors/string_executor_test.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license
+ * Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
update to 2019 for all files, thanks.

Reply:
updated (https://github.com/tensorflow/tfjs-converter/pull/376/commits/c9ba5f63ee3ff2555b867f903e85d6c07b5771b8, https://github.com/tensorflow/tfjs-converter/pull/376/commits/fb065c7b3ff7f68809b0e4c8ae632cefb836b8f7)